_content/papers/buddha-and-metaphysics_wijesekera.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Buddha and Metaphysics"
+authors:
+  - wijesekera
+editor: "M. H. F. Jayasuriya"
+external_url: "https://archive.org/details/in.ernet.dli.2015.463583/page/n3/mode/2up"
```

Comment:
This cut of the PDF is unique to OBU, so I uploaded it to our internal storage. Link to it like so instead:

```yaml
file_links:
  - "exclusive_01/The%20Buddha%20and%20Metaphysics%20-%20Wijesekera.pdf"
```